.github/workflows/cd-ec2.yml

```
@@ -36,13 +38,34 @@ jobs:
         with:
-          host: ${{ secrets.EC2_HOST }}
+          host:     ${{ secrets.EC2_HOST }}
           username: ${{ secrets.EC2_USER }}
-          key: ${{ secrets.EC2_SSH_KEY }}
+          key_path: key.pem      # key 대신 key_path 사용 권장
+          port: 22
           script_stop: true
           script: |
             docker login ghcr.io -u ${{ github.actor }} -p ${{ secrets.GHCR_TOKEN }}
-            bash /home/ec2-user/deploy.sh
+            bash /home/ec2-user/deploy.sh
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)